### PR TITLE
server: use large heartbeat and election

### PR DESCRIPTION
**Pull request by siddontang**
Now the heartbeat is 1s and election timeout is 6s. ( 5 * heartbeat > election in etcd )
But we must know that if the PD is down, we will begin the election after 6s later. Is it too long?
Or maybe we can use 500ms and 3s instead? 

@ngaut @qiuyesuifeng @huachaohuang @overvenus

### CODE REVIEW

**Comment by huachaohuang** [comment]
Why should we use large heartbeat interval and election timeout?

File: server/config.go [hunk 1]
```diff
@@ -225,6 +234,9 @@ func (c *Config) adjust() error {
 		c.nextRetryDelay = defaultNextRetryDelay
 	}
 
+	c.tickMs = defaultTickMs
```
Comment (huachaohuang) [inline server/config.go]: This will overwrite the `c.tickMs`, no matter it is configured or not.

**Comment by huachaohuang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @ngaut

**Comment by ngaut** [comment]
LGTM

**Pull request merged**